### DISCUSSION TITLE
How to support x64 & arm64 servers simultaneously?

### DISCUSSION THREAD

**Discussion by 13126690189** [cobbler/cobbler#2778] [Q&A]
I want to use a single cobbler server to install OS on both types of servers.  Is that possible? and how to do that?

**Comment by SchoolGuy** [comment]
Yes it is. You just need to have multiple distros and bootloaders presents for the corresponding architectures. On openSUSE/SLE the GRUB Bootloaders are packaged as noarch so you can install them from any arch onto any arch. Thus in combination with the `mkgrub.sh` script you have the scenario done nearly out of the box.
